code/modules/gear_presets/dutch.dm

```diff
@@ -0,0 +1,244 @@
+//Dutch's Dozens, pred hunter ERT, moved from fun file as they don't fit there to be honest.
+
+/datum/equipment_preset/dutch
+	name = JOB_DUTCH_RIFLEMAN
+	paygrade = "DTC"
+	assignment = JOB_DUTCH_RIFLEMAN
+	flags = EQUIPMENT_PRESET_EXTRA
+	faction = FACTION_DUTCH
+
+	skills = /datum/skills/dutchmerc
+
+/datum/equipment_preset/dutch/New()
+	..()
+	rank = assignment
+
+/datum/equipment_preset/dutch/load_name(mob/living/carbon/human/H, randomise)
+	H.gender = pick(60;MALE,40;FEMALE)
```

Comment:
use the pick_weight proc instead of dm built-in pickweight

Reply:
I don't know how to(that code is esoteric), and it is used 0(zero) times in the code.